### ISSUE TITLE
Why do you assume that all target meshes are female in SURREAL?

### ISSUE THREAD

**Issue by hongsukchoi** [zengwang430521/DecoMR#5]
I run your evaluation code on SURREAL test set (493 imgs), and found out that the dataloader code only provides __female__ genders.
Why don't you use the groundtruth genders which are provided in the dataset for the correct evaluation? 

When I test your pre-trained model on SURREAL test set with groundtruth genders, I get 
__MPJPE: 67.82__
__MPVPE (Surface error): 64.59__  

These numbers are far different from the numbers reported in the paper (__MPVPE: 56.5__ , Tab.3) and not much good compared to the accuracy of  BodyNet (__MPVPE (Surface error): 65.8__).

When I test your pre-trained model on SURREAL test set with intact code, I get
__MPJPE: 54.29__
__MPVPE (Surface error): 51.65__  

Do you have any idea to get the numbers you reported in the paper?
And I think assuming all target meshes as female is inappropriate and unfair to compare to BodyNet results.

**Comment by zengwang430521** [comment]
Hi, thanks for your interest in our work. I'm sorry for my late reply.

I'm sorry that there is a bug in my code of SURREAL dataset. In SURREAL, the gender is represented by 0 and 1, while in 3DPW the gender is represented by 'm' and 'f'. And I made a mistake in this issuse.

I will fix this bug and retrain a model in a few days.

**Comment by zengwang430521** [comment]
Hi, we have fixed the bug on SURREAL dataset and retrain a model on SURREAL dataset. You can update the code and annotation files of SURREAL following  the readme and get the retrained model fro here ([https://drive.google.com/drive/folders/1xWBVfQa7OZ14VgT9BVO9Lj_kDqRAcQ-e?usp=sharing).

**Comment by hongsukchoi** [comment]
Thank you for your effort! I will look into that
